langlib/lang.decimal/src/main/ballerina/decimal.bal

```diff
@@ -66,6 +110,20 @@ public isolated function abs(decimal x) returns decimal = @java:Method {
 # roundToIntegralTiesToEven will return a value with a positive exponent when the operand has a positive exponent.
 # Note that `<int>x` is the same as `<int>x.round(0)`.
 #
+# ```ballerina
+# decimal d = 3.55;
+# d.round() ⇒ 4
+#
+# decimal e = 4.55555;
+# e.round(3) ⇒ 4.556
+# 
+# decimal g = 2.5;
+# g.round(0) ⇒ 2
+# 
+# decimal h = 3.5;
+# h.round(0) ⇒ 4
+# ```
```

Comment:
Shall we also add an example with a negative value for `fractionDigits`?

Reply:
yes, will add, it is better.